src/QueryBuilder.php

```diff
@@ -49,8 +56,8 @@ public static function create(): self
 
     public function getSearchParams(string $queryString, string $type, int $start, int $count, string $order, string $dir): array
     {
-        $qs = htmlspecialchars_decode($queryString);
-        $blocks = $output = preg_split( '/( AND )/', $qs);
+        $decoded_query_string = htmlspecialchars_decode($queryString);
```

Comment:
Is this the correct place to decode the string?

Reply:
I think it's ok to be here or I can move it to ElasticSearch->query() method that calls it, but it won't change much I think